pkg/operation/botanist/component/etcdcopybackupstask/etcdcopybackupstask_test.go

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcdcopybackupstask_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/gardener/gardener/pkg/logger"
+	mockclient "github.com/gardener/gardener/pkg/mock/controller-runtime/client"
+	. "github.com/gardener/gardener/pkg/operation/botanist/component/etcdcopybackupstask"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+var _ = Describe("EtcdCopyBackupsTask", func() {
+	var (
+		ctrl *gomock.Controller
+		ctx  context.Context
+		c    *mockclient.MockClient
+		log  logrus.FieldLogger
+
+		expectedTaskResource *druidv1alpha1.EtcdCopyBackupsTask
```

Comment:
I think it's better to use a shorter and simpler name that also matches the name used in the code.
```suggestion
		task *druidv1alpha1.EtcdCopyBackupsTask
```